docs/logagent/output-filter-geoip.md

```diff
@@ -0,0 +1,90 @@
+Title: Add Geo-IP information to logs 
+Description: Add Geo-IP information to logs. log enrichment 
+
+
+## Output filter: geoip
+
+This plugin adds Geo IP information to logs. 
+An everyday use case is to enrich web server logs (or any logs with IP addresses) with geographical information derived from those IP addresses. 
+ 
+Things you do not need to think about at all:
+
+- Maxmind Geo-IP lite database is downloaded automatically to /tmp/
+  To change the location of the DB set MAXMIND_DB_DIR=path
+- Integrated automatic updates for the GeoIP database. 
+  The update check runs every hour. 
+- Elasticsearch mapping for the Geo-Coordinates in Sematext Logs for geographic queries and map displays. Sematext Logs indices support the `geoip` field out of the box. 
+
+
+
+### Configuration 
+
+Here is how to enable Geo IP lookups for your logs:
```

Comment:
Change this to a list, instead of using 1), 2) etc...
Just make it a list:

1.
2. 
3. 
...

And Markdown will render it like a list.